error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/StreamRulesTestInput.java

```diff
@@ -1,10 +1,13 @@
 package tech.picnic.errorprone.refasterrules;
 
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
```

Comment:
W.r.t. your question about how to handle those unused imports: for this we have the [`elidedTypesAndStaticImports`](https://github.com/PicnicSupermarket/error-prone-support/blob/57fa6ae2b830be9b8b67f6fb55105c59fe717244/error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/StreamRulesTestInput.java#L37-L58) method: by referencing the type or member there (in both input and output file) the import won't become unused.

Reply:
Makes sense, thanks for the explanation!